installers/cubeit-installer

```diff
@@ -80,6 +80,7 @@ cat << EOF
     --privilegedcn: set container name for privileged
     --encrypt: encrypt the rootfs
     --ima-sign: sign the files with IMA signature
+    --yes: do not ask questions
```

Comment:
This should be "answer 'yes' to all questions'.

If it is "ask no questions", the mode should be 'silent' 

But even this isn't quite enough, since with that short message, the user has no idea what they are answering yes to.
So this option needs a longer help text that summarizes what 'yes' will be used for.